### PR TITLE
Added tooltip to text box input

### PR DESCRIPTION
**Pull request by mar257**
# Problem

Incorporate bootstrap tooltip to text box 
# Solution

Incorporated tooltip on bottom that triggers on hovering over the text box and says "This is the new text that we will add to the list!"

### CODE REVIEW

**Comment by FreekingDean** [comment]
👍  just one comment.

File: App.js [hunk 1]
```diff
@@ -38,14 +38,16 @@ class App extends React.Component {
   render(){
     return (
       <div>
-        <input type="textarea" name="textToAdd" id="textToAdd" size="75" defaultValue={this.props.text}/>
-        <input type="submit" value="Add" id="addbutton" onClick={this.update.bind(this)}/>
+        <input type="textarea" name="textToAdd" id="textToAdd" size="75" defaultValue={this.props.text} data-toggle="tooltip" data-placement="bottom" title="This is the new text we will add to the list!"/>
+        <input type="button" value="Add" id="addbutton" onClick={this.update.bind(this)}/>
```
Comment (FreekingDean) [inline App.js]: is there a reason you changed this from a `submit` to a `button`?

Comment (mar257) [inline App.js]: No, honestly, I did that when I was playing around with it and never changed it back

Comment (FreekingDean) [inline App.js]: Ah! I would suggest reverting it to the way it is in `master` this way there are no merge conflicts 😃

Comment (mar257) [inline App.js]: Excellent point, I see now, I'll change it back

File: App.js [hunk 2]
```diff
@@ -30,6 +30,13 @@ class App extends React.Component {
       ul.appendChild(li); //append li to list
     }
   }
+
+
+  makeListItem(textToAdd) {
```
Comment (FreekingDean) [inline App.js]: Where is this method used?

File: App.js [hunk 3]
```diff
@@ -31,6 +31,12 @@ class App extends React.Component {
     }
   }
 
+  makeListItem(textToAdd) {
```
Comment (FreekingDean) [inline App.js:34]: Where is this method used? (I'm guessing it was from the list item tooltip stuffs?)

Comment (mar257) [inline App.js:34]: Yeah it was part of an attempt to incorporate the list tooltip functionality, and I forgot to get rid of it, hence all of these problems

**Pull request merged**